### PR TITLE
STONK-4: Dependencies > Migrate Package Manager To PNPM

### LINKED ISSUE #5: STONK-4: Dependencies > Migrate Package Manager To PNPM
## Description

We want to migrate to a more performant and space-efficient package manager like [PNPM](https://pnpm.io/) .

## Acceptance Criteria

- move from npm as our package manager to pnpm
- running `pnpm i` should generate a `pnpm-lock.yaml` in both repos

## Checklist

- [x] I have verified that this issue is relevant to the current project scope
- [x] I have added the appropriate label to this issue
- [x] I have linked this issue to the project

### PR DESCRIPTION
**Pull request by Jenna59**
## Ticket

Closes #5 

## Changes

Installed pnpm. Changed several frontend react-router imports to react-router-dom as there was a vite error re: react-router not being found.

## Developer Checklist

- [Y] I have verified my changes to confirm that it works
- [Y] I have added the appropriate label to this pull request
- [Y] I have linked the corresponding issue to this pull request
- [Y] I have followed the pull request title pattern of `STONK-NUMBER: Domain > Issue Title`

### CODE REVIEW

**Review by JackyTea — APPROVED** [review]

File: backend/package.json [hunk 1]
```diff
@@ -17,6 +17,7 @@
     "express": "^4.17.1",
     "http": "*",
     "jsonwebtoken": "^9.0.2",
+    "mongodb": "^6.2.0",
```
Comment (JackyTea) [inline backend/package.json:20]: Why was this added to dependencies?

Comment (Jenna59) [reply backend/package.json:20]: No. I think I just got confused by the documentation about the Node driver in step #5: https://www.mongodb.com/docs/atlas/getting-started/ and thought that was what I needed to have it connect properly. I don't think I've ever worked with Mongoose. I will remove the package.

Comment (JackyTea) [reply backend/package.json]: No worries. Thanks for reverting it. The rest looks good.

**Review by JackyTea — APPROVED** [review]
